### PR TITLE
feat(docs): animate header logos when switching

### PR DESCRIPTION
**Pull request by JohnPhamous**
https://user-images.githubusercontent.com/16027268/211107001-748f516b-77ca-4a11-a4d6-d314ab5db454.mp4

### CODE REVIEW

**Comment by ForsakenHarmony** [comment]
Looks like the logo is slightly "overshooting", not a big fan of that, but 🤷‍♀️

**Comment by JohnPhamous** [comment]
> Looks like the logo is slightly "overshooting", not a big fan of that, but 🤷‍♀️

Are you referring to this overlap? 

If yeah then same, I'll fix!

CleanShot 2023-01-06 at 14 04 25@2x

File: docs/components/logos/TurboAnimated.tsx [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+import { useRouter } from "next/router";
+
+type LogoProps = {
+  className?: string;
+  height?: number;
+};
+
+const TurboAnimated = ({ height = 32, className = "" }: LogoProps) => {
+  const router = useRouter();
+  const showTurboRepoLogo = !router.route.startsWith("/pack");
```
Comment (tknickman) [inline docs/components/logos/TurboAnimated.tsx:11]: You can `useTurboSite` (from [here](https://github.com/vercel/turbo/blob/main/docs/components/SiteSwitcher.tsx#L5))

**Review by tknickman — COMMENTED** [review]
Only one code suggestion, looking over the preview now!

**Review by tknickman — APPROVED** [review]
Looks 🔥

**Pull request merged**